integration/common/frontend.py

```diff
@@ -10,7 +10,7 @@
 
 
 def readat_direct(dev, offset, length):
-    pg = offset / PAGE_SIZE
+    pg = offset // PAGE_SIZE
```

Comment:
Why changed this?

Reply:
I think we want to get page count here, so it's better to get an integer. Another line in `writeat_direct` also uses `//`.

https://github.com/longhorn/longhorn-engine/blob/c672c3a30435e50d3922b739ac274e15c7e19da1/integration/common/frontend.py#L33-L34

Reply:
cc @shuo-wu

Reply:
As discussed, please help create a ticket to review these read/write direct functions later.

Reply:
We got error here because we upgrade Python version. In new version, `os.lseek` needs input type as integer.

```python
(function) def lseek(
    __fd: int,
    __position: int,
    __whence: int,
    /
) -> int
```

```
    def readat_direct(dev, offset, length):
        pg = offset / PAGE_SIZE
        in_page_offset = offset % PAGE_SIZE
        # either read less than a page, or whole pages
        if in_page_offset != 0:
            assert pg == (offset + length) / PAGE_SIZE
            to_read = PAGE_SIZE
        else:
            assert length % PAGE_SIZE == 0
            to_read = length
        pg_offset = pg * PAGE_SIZE
    
        f = os.open(dev, os.O_DIRECT | os.O_RDONLY)
        try:
>           os.lseek(f, pg_offset, os.SEEK_SET)
E           TypeError: 'float' object cannot be interpreted as an integer
```

Reply:
The change is OK, as it's looking for the start page to write and read.